### PR TITLE
AGENT-948: day2 pxe support

### PR DESCRIPTION
**Pull request by zniu1011**
Add agent day2 pxe support flow and jobs for 4.18/4.19

### CODE REVIEW

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-pxe-compact-agent-ipv4-static-day2-disc-mixarch-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-pxe-compact-agent-ipv4-static-day2-disc-mixarch-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-multi-nightly-baremetal-pxe-compact-agent-ipv4-static-arm64-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.19-arm64-nightly-baremetal-pxe-compact-agent-ipv4-static-day2-f14

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly.yaml [hunk 1]
```diff
@@ -1886,6 +1886,27 @@ tests:
...
+  cluster: build05
+  cron: 4 17 8,24 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: x86_64
+      ADDITIONAL_WORKERS: "2"
+      AGENT_PLATFORM_TYPE: none
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      E2E_RUN_TAGS: '@disc'
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly.yaml:1899]: Is the tag the expected one ?

**Pull request merged**